app/views/issues/map.html.slim

```diff
@@ -0,0 +1,54 @@
+link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css"
+script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"
+script src="http://www.matchingnotes.com/javascripts/leaflet-google.js"
+
+h1 Мапа звернень
+#issues-map
+
+javascript: 
+  $(document).ready(function(){
+    var map = new L.Map('issues-map', 
+              { center: new L.LatLng(50.1192478, 26.8197633), 
+                zoom: 9});
+    var googleLayer = new L.Google('ROADMAP');
+    map.addLayer(googleLayer);
+    
+    $.get({url: '/map', dataType:"json", cache:false}).done(function(issues) { 
+      addMarkers(issues); 
+    });
+
+    function addMarkers(issues) {
+      var bounds = [];
+      issues.forEach(function(issue){
+        var coords = [issue.latitude, issue.longitude];
+        var marker = L.marker(coords).addTo(map);
+        bounds.push(coords)
+        handlePopup(marker, issue);
+      });
+      map.fitBounds(bounds);
+    }
+    
+    function handlePopup(marker, issue){
+      marker.bindPopup("Завантаження...")
+      marker.on('click', function(e) {
+          var popup = e.target.getPopup();
+          var url="/map/issue/" + issue.id ;
+          $.get({url: url, dataType:"json"}).done(function(response) {
+            var shortDescription = truncate.apply(response.issue.description, [90, true])
+            var popupLayout = '<h4>'+response.issue.title+'</h4>'+
+                              '<img src="'+ response.img.attachment.url +'">'+
+                              '<p>'+shortDescription+'</p>'+
+                              '<p class="text-right"><a href="/issues/'+response.issue.id+'">Переглянути</a></p>';
+            popup.setContent(popupLayout);
+            popup.update();
+          });
+      });
+    }
+
+    function truncate(n, useWordBoundary){
```

Comment:
Why create impure function and use `this` keyword? Better send word that should be truncated explicitly as parameter.
function truncate(word, maxLenght, useWordBoundary)